config/db.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose')
+const config = require('config')
+const db = config.get('mongoURI')
+
+const connectDB = async () => {
+  try {
+    await mongoose
+    .connect(db, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true
+    })
```

Comment:
We should prob add semicolons throughout our code, even though they aren't necessary, they help with readability. Up to you though. ESLint _should_ pick this up if it's configured for this project.

Reply:
Done, using Prettier.